security-proxy/src/deb/resources/etc/georchestra/security-proxy/security-proxy.properties

```diff
@@ -0,0 +1,26 @@
+proxy.defaultTarget=http://georchestra.mydomain.org/
+public.host=http://georchestra.mydomain.org/
+anonymousRole=ROLE_ANONYMOUS
+proxy.contextPath=/sec
+logout-success-url=https://georchestra.mydomain.org/cas/logout?fromgeorchestra
+casLoginUrl=https://georchestra.mydomain.org/cas/login
+casTicketValidation=https://georchestra.mydomain.org/cas
+proxyCallback=https://georchestra.mydomain.org/j_spring_cas_security_check
+ldapUrl=ldap://localhost
+baseDN=dc=georchestra,dc=org
+userSearchBaseDN=ou=users
+userSearchFilter=(uid={0})
+authoritiesBaseDN=ou=groups
+groupRoleAttribute=cn
+groupSearchFilter=(member=uid={1},ou=users,dc=georchestra,dc=org)
+ldapAdminDn=cn=admin,dc=georchestra,dc=org
+ldap.admin.password=secret
+checkHealth=true
+psql.host=localhost
+psql.port=5432
+psql.db=georchestra
+psql.user=www-data
+psql.pass=www-data
+max.database.connections=10
+realmName=georchestra
```

Comment:
Not a big deal, but we lost all comments from https://github.com/georchestra/georchestra/blob/master/security-proxy/src/main/filtered-resources/WEB-INF/security-proxy.properties ...